Zeplin.sketchplugin/Contents/Sketch/export.cocoascript

```diff
@@ -138,6 +138,22 @@ var exportColors = function (context) {
         return;
     }
 
+    var assets = context.document.documentData().assets();
+    var colors;
+
+    // `colors` is deprecated at Sketch version 53+
+    if (assets.colors) {
+        colors = assets.colors();
+    } else {
+        colors = assets.colorAssets();
+    }
+
+    if (!colors || !colors.length) {
+      [NSApp displayDialog:@"No colors found in document" withTitle:@"Cannot export colors"];
```

Comment:
I'm open to copy suggestions. 😅 

Wdyt about adding a button (`Help`) to redirect user to a support article about the document colors/text styles?